Documentation/Contributing.md

```diff
@@ -0,0 +1,160 @@
+* If during `swift build` you encounter this error:
+```
+/../apple-repos/swift-package-manager/.build/checkouts/swift-driver/Sources/SwiftDriver/Explicit Module Builds/InterModuleDependencyGraph.swift:102:3: error: unknown attribute '_spi'
+  @_spi(Testing) public var isFramework: Bool
+  ^
+```
+Make sure you are using SwiftPM 5.3
+```
+$> swift package --version
+Swift Package Manager - Swift 5.3.0
+```
+* If during `swift build` you encounter this error:
+```
+/../swift-package-manager/Sources/PackageLoading/Target+PkgConfig.swift:84:36: error: type 'PkgConfigError' has no member 'prohibitedFlags'
+            error = PkgConfigError.prohibitedFlags(filtered.unallowed.joined(separator: ", "))
+                    ~~~~~~~~~~~~~~ ^~~~~~~~~~~~~~~
+```
+Make sure to update your TSC (Tools Support Core):
+```
+swift package update
+```
+Alternatively, if you are using Xcode, you can update to the latest version of all packages:  
+**Xcode App** > *File* > *Swift Packages* > *Update to Latest Package Versions*
+
+### Find your way to contribute
+Report a bug guide: https://github.com/apple/swift-package-manager/blob/main/Documentation/Resources.md#reporting-a-good-swiftpm-bug.  
+JIRA Bug Tracker (a place where you can open bugs, enhancements to start to contribute): [https://bugs.swift.org/browse/SR-13640?jql=component%20%3D%20%22Package%20Manager%22](https://bugs.swift.org/browse/SR-13640?jql=component%20%3D%20%22Package%20Manager%22).
+
+### Commits / PRs
+1. Fork: https://github.com/apple/swift-package-manager
+2. Clone a working copy of your fork
+3. Create a new branch
+4. Make your code changes
+5. Commit (include the Radar link or JIRA issue id in the commit message if possible and a description your changes)
+6. Push the commit / branch to your fork
+7. Make a PR from your fork / branch to `apple: main`
+8. Leave a new comment to trigger smoke tests: `@swift-ci please smoke test`
+9. Reviewers are going to be automatically added to your PR
+10. Merge pull request when you received approval from the reviewers (one or more)
+
+### Community and Support
+If you want to connect with the Swift community you can:
+* Use Swift Forums: [https://forums.swift.org](https://forums.swift.org/)
+* Contact the CODEOWNERS: https://github.com/apple/swift-package-manager/blob/main/CODEOWNERS
+(mailing lists are usually the best place to go for help: [code-owners@swift.org](mailto:code-owners@swift.org), [conduct@swift.org](mailto:conduct@swift.org), [swift-infrastructure@swift.org](mailto:swift-infrastructure@swift.org)
+
+### Additional Links
+* Official Apple GitHub
+[https://github.com/apple](https://github.com/apple)
+* Swift Package Manager GitHub
+[https://github.com/apple/swift-package-manager](https://github.com/apple/swift-package-manager)
+* Setup Development
+[https://github.com/apple/swift-package-manager/blob/main/Documentation/Development.md](https://github.com/apple/swift-package-manager/blob/main/Documentation/Development.md)
```

Comment:
is there anything in Development.md that this doc does not cover? If not we should remove Development.md otherwise we should bring in the missing pieces and remove Development.md

Reply:
@federicobucchi was ^^ adressed?

Reply:
I brought everything that was missed from Development.md and I removed Development.md